doc/integrator/docker.rst

```diff
@@ -6,8 +6,15 @@ Use Docker to deploy your application
 Architecture schema
 -------------------
 
-.. image:: docker.png
-.. source file is docker.dia.
+For OpenShift projects:
```

Comment:
There not big changes but to don't set some wrong things I prefer to have tow diagram them one too complex...
What do you purpose?

Reply:
Yes, two diagrams is better than one here.
But we don't have any other information on OpenShift in our docu, so this diagram comes as a surprise, and some additional explanation would be cool (but it can also be done in a later PR)